build/script.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
```

Comment:
It's my solution to this problem: http://stackoverflow.com/questions/41398737/karma-testing-a-bundled-file

In future this will be fixed with a [rollup plugin](https://github.com/shouston3/rollup-plugin-init), but that is still in progress